lib/tapjoy/ldap/cli/group/delete.rb

```diff
@@ -0,0 +1,37 @@
+module Tapjoy
+  module LDAP
+    module CLI
+      module Group
+        # Delete LDAP group
+        class Delete
+          def delete
+            confirm unless opts[:force]
+            puts Tapjoy::LDAP::API::Group.destroy(opts[:name])
+          end
+
+          private
+          
+          def opts
+            @opts ||= Trollop.options do
+              # Set help message
+              usage 'group delete [options]'
+              synopsis "\nThis command is for deleting LDAP groups"
+
+              opt :name, 'Specify group', type: :string, required: true
+              opt :force, 'Force delete'
+            end
+          end
+
+          def confirm
+            puts "Confirm that you want to delete group #{opts[:name]} (yes/no)"
+            print '>'
+            confirm = STDIN.gets.chomp().downcase
+            unless confirm.eql?('y') || confirm.eql?('yes')
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use a guard clause instead of wrapping the code inside a conditional expression.](https://www.codacy.com/app/ali-tayarani/ldap_tools/file/2634622856/issues/source?bid=3316970&fileBranchId=3318857#l29)